README.md

```diff
@@ -191,7 +191,7 @@ If you would you like to start your own language implementation of IPFS, check o
 | C++ | https://github.com/vasild/cpp-ipfs-api | |
 | Julia | contact: [@rened](https://github.com/rened) | 0% |
 | Lua | contact: [@seclorum](https://github.com/seclorum) | 0% |
-| Erlang | ! | 0% |
+| Erlang | https://github.com/hendry19901990/erlang-ipfs-api |  |
```

Comment:
The Manual is Ready!, Check it out please https://github.com/hendry19901990/erlang-ipfs-api/blob/master/README.md